README.md

```diff
@@ -16,6 +18,9 @@ Follow these steps to open this sample in a Codespace:
 For more info, check out the [GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/developing-online-with-codespaces/creating-a-codespace#creating-a-codespace).
 
 ### VS Code Remote - Containers
+
+If you already have Docker installed, then you may use [open in dev container](https://insiders.vscode.dev/redirect?url=vscode-insiders://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/microsoft/vscode-remote-try-java) (which provides the same functionality as the Remote - Containers badge above). This installs the Remote - Containers extension if needed, clones the source code in a Docker Volume, and opens it in VS Code.

```

Comment:
Possible language tweaks after reading this:

```suggestion
If you already have VS Code and Docker installed, you can click the badge above or [here](https://insiders.vscode.dev/redirect?url=vscode-insiders://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/microsoft/vscode-remote-try-java) to get started. Clicking these links will cause VS Code to automatically install the Remote - Containers extension if needed, clone the source code into a container volume, and spin up a dev container for use.

```

Reply:
👍 on the wording tweaks.